src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -94,16 +94,16 @@
     );
 
     private String[] capFiles = {"AArch64LibCHelperDirectives.cap",
-            "AMD64LibCHelperDirectives.cap", "BuiltinDirectives.cap",
-            "JNIHeaderDirectives.cap", "LibFFIHeaderDirectives.cap",
-            "PosixDirectives.cap"};
+        "AMD64LibCHelperDirectives.cap", "BuiltinDirectives.cap",
+        "JNIHeaderDirectives.cap", "LibFFIHeaderDirectives.cap",
+        "PosixDirectives.cap"};
 
     private String llvmCapFile = "LLVMDirectives.cap";
 
-    private final String capLocation= "/native/linux-aarch64/cap/";
+    private final String capLocation = "/native/linux-aarch64/cap/";
 
     private static final List<String> linuxfxSWlibs = Arrays.asList(
-            "-Wl,--whole-archive", "-lprism_sw", "-Wl,--no-whole-archive", "-lm");
+            "-Wl,--whole-archive", "-lprism_sw", "-Wl,--no-whole-archive");
```

Comment:
yes, that always needs to be there, in the end. It might not always be required (although it probably is now that we use pow in launcher.o), but in that case it doesn't hurt.